### PR TITLE
Better Python Notes

### PR DESCRIPTION
**Pull request by Sam-Varghese**
Respected sir/mam.
Myself Sam Varghese, I am a grade 12th student and a qualifier of microsoft 98-381 exam which is for python. I saw these notes  and found these notes a bit incomplete for python. Hence  I worked hard in putting various important points and extra informations that a programmer usually forgets ,but should always know .I have updated these notes with useful concepts of grade 11th and 12th. Following are the changes that I made

I have:-
Put logo
Put font-styles,colors
Put extra important points
Put precedence order of operators
Put scope of variables LEGB Rule
Put list methods
Put dictionary methods
Put concepts of text file
Put concepts of csv file
Put concepts of binary files

Hence I request you to accept my pull request . In case of any suggestions , further improvements , please do let me know. I would be waiting for your useful feedbacks.

Thankyou 
Sam Varghese

### CODE REVIEW

File: languages/Python/dictionary_methods.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# <div style='text-align:center'><span style='color:gold;font-family:georgia'>Dictionary Methods
```
Comment (Sam-Varghese) [inline languages/Python/dictionary_methods.md:1]: This would beautify our notes

**Comment by LeCoupa** [comment]
Hello @Sam-Varghese , sorry I can't accept the PR as it does not follow the one-file cheatsheet convention we use.

**Comment by Sam-Varghese** [comment]
Ok no problem ,thanks for reviewing ,would try to meet your conventions for making notes next time.

**Comment by LeCoupa** [comment]
Sorry about that :) Have a good day

**Pull request closed without merging**